graylog2-server/src/main/java/org/graylog2/shared/security/RestPermissions.java

```diff
@@ -245,6 +263,8 @@
         BUFFERS_READ,
         CLUSTER_CONFIG_ENTRY_READ,
         DECORATORS_READ,
+        EVENT_DEFINITIONS_READ,
+        EVENT_NOTIFICATIONS_READ,
```

Comment:
Why do reader users need these permissions? I think we should leave them out for now until we know how to proceed with alerting permissions.

Reply:
Oh, I thought the Reader role was meant to be a reviewer, not the default for every user.
Will change that.

Reply:
Oh, I thought the Reader role was meant to be a reviewer, not the default for every user.
Will change that.